Common/include/geometry/CMultiGridGeometry.hpp

```diff
@@ -61,101 +45,159 @@ class CMultiGridGeometry final : public CGeometry {
    * \param[in] config - Definition of the particular problem.
    * \return <code>TRUE</code> or <code>FALSE</code> depending if the control volume can be agglomerated.
    */
-  bool SetBoundAgglomeration(unsigned long CVPoint, short marker_seed, CGeometry *fine_grid, CConfig *config);
+  bool SetBoundAgglomeration(unsigned long CVPoint, short marker_seed, const CGeometry *fine_grid,
+                             const CConfig *config) const;
 
   /*!
    * \brief Determine if a can be agglomerated using geometrical criteria.
    * \param[in] iPoint - Seed point.
    * \param[in] fine_grid - Geometrical definition of the problem.
    * \param[in] config - Definition of the particular problem.
    */
-  bool GeometricalCheck(unsigned long iPoint, CGeometry *fine_grid, CConfig *config);
+  bool GeometricalCheck(unsigned long iPoint, const CGeometry *fine_grid, const CConfig *config) const;
```

Comment:
Maybe with that amount of changes clang-format could have been a thing. (Again pointer-Alignment is the rat here)